iocage/lib/JailConfigResolver.py

```diff
@@ -0,0 +1,89 @@
+import helpers
+
+import shutil
+
+
+class JailConfigResolver(list):
+
+    def __init__(self, jail_config, logger=None):
+        list.__init__(self, [])
+        helpers.init_logger(self, logger)
+        self.jail_config = jail_config
+        self.jail_config.update_special_property(
+            "resolver", new_property_handler=self)
+
+    @property
+    def conf_file_path(self):
+        return "/etc/resolv.conf"
+
+    @property
+    def method(self):
+        if self.value == "/etc/resolv.conf":
+            return "copy"
```

Comment:
I'm saying if you gonna use those values in other places in the same file and those won't be changed you can try to make something static using a tuple, something like: result = ('copy', 'manual', 'skip').

But this is not mandatory and is more cosmetic than anything else. It is all up to you :).